lib/tasks/migrate/20250817055243_remove_blank_media_items.rake

```diff
@@ -74,13 +76,28 @@ namespace :check do
         ProjectMedia.select('project_medias.*, fact_checks.title AS fc_title')
         .where(id: pm_ids)
         .joins({claim_description: :fact_check})
-        .find_in_batches(batch_size: 500) do |items|
+        .find_in_batches(batch_size: 1000) do |items|
           print '.'
+          # Define mapping array for quote => pm_id
+          pm_quote_mapping = {}
+          claim_values = []
           media_ids = items.pluck(:media_id)
           items.each do |pm|
             print '.'
-            claim = Claim.create!(quote: pm.fc_title)
-            pm.update_column(:media_id, claim.id)
+            pm_quote_mapping[Digest::MD5.hexdigest(pm.fc_title)] = pm.id
+            claim_values << { type: 'Claim', quote: pm.fc_title, user_id: pm.user_id, created_at: pm.created_at, updated_at: pm.updated_at }
+          end
+          result = Claim.insert_all(claim_values, returning: [:id, :quote])
+          project_media_values = []
+          result.rows.each do |claim_raw|
+            id, quote = claim_raw
+            # Set media_id
+            pm_id = pm_quote_mapping[Digest::MD5.hexdigest(quote)]
+            project_media_values << { id: pm_id, media_id: id }
+          end
+          # Bulk update ProjectMedia
+          project_media_values.each do |r|
+            ProjectMedia.where(id: r[:id]).update_all(media_id: r[:media_id])
           end
           # Destroy Blank items
           Media.where(id: media_ids).delete_all
```

Comment:
Better to be super-safe and double check that `type` here is `Blank`.

Reply:
Done